AEPIdentity/Sources/IdentityProperties.swift

```diff
@@ -63,6 +63,24 @@ struct IdentityProperties: Codable {
         return eventData
     }
 
+    /// Converts `IdentityProperties` into an XDM data representation
+    /// - Returns: A dictionary representing this `IdentityProperties` in XDM data format
+    func toXDMData() -> [String: Any] {
+        var identityMap = IdentityMap()
+
+        // Add ECID
+        if let ecidString = ecid?.ecidString {
+            identityMap.addItem(namespace: IdentityConstants.XDM.Keys.ECID, id: ecidString)
+        }
+
+        for customerId in customerIds ?? [] {
+            guard let type = customerId.type, let id = customerId.identifier else { continue }
+            identityMap.addItem(namespace: type, id: id, authenticationState: customerId.authenticationState, primary: false)
```

Comment:
This looks good for now for customer visitor ids. We are still confirming with the services team that this is supported for all customer id name, but we can revise this later when we have more details.

We will need to handle IDFA differently, which is currently stored in the customerIds list.